packages/rebber-plugins/src/type/gridTable.js

```diff
@@ -112,7 +114,7 @@ class GridTableStringifier {
       this.nbOfColumns = this.colIndex
     }
     this.colIndex = 0
-    return rowText
+    return rowText.replace(/\\number-of-column/, this.nbOfColumns)
```

Comment:
This should only replace the first occurrence. Is this really what you meant? If so please remove the regex.

Reply:
At the begining it was not what I needed, I just forgot the `g` option.

But in fact, this can only be found on the first row so I don't know if the regexp is really the right thing to do